From 0f6214894f4266ec5de311ab8637c86f2258e2b9 Mon Sep 17 00:00:00 2001
From: Jason Liu <jason.hui.liu@nxp.com>
Date: Mon, 29 Oct 2018 10:50:30 +0800
Subject: [PATCH 0002/5242] Revert "drm/imx: imx-ldb: disable LDB on driver
 bind"

commit  8f43a447bcdc54a8a4949f7dc79fc25732c982f1 from
https://source.codeaurora.org/external/imx/linux-imx.git

This reverts commit a43eac2d855be9f441b0c6e277bdaf878b07a370.
This patch break the i.MX8, revert it firstly.
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/imx-ldb.c |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/imx-ldb.c b/drivers/gpu/drm/imx/imx-ldb.c
index dd5312b..3f02ee5 100644
--- a/drivers/gpu/drm/imx/imx-ldb.c
+++ b/drivers/gpu/drm/imx/imx-ldb.c
@@ -612,9 +612,6 @@ static int imx_ldb_bind(struct device *dev, struct device *master, void *data)
 		return PTR_ERR(imx_ldb->regmap);
 	}
 
-	/* disable LDB by resetting the control register to POR default */
-	regmap_write(imx_ldb->regmap, IOMUXC_GPR2, 0);
-
 	imx_ldb->dev = dev;
 
 	if (of_id)
-- 
1.7.9.5

